src/controller/game_controller.py

```diff
@@ -75,18 +89,49 @@ def playOneRound(self):
             # Let human choose
             state = True
             while state:
-                for event in pygame.event.get():
-                    if event.type == QUIT:
-                        return False
-                    elif event.type == KEYDOWN and event.key == K_ESCAPE:
+                event = pygame.event.wait()
+                if event.type == QUIT:
+                    return False
+
+                elif event.type == KEYDOWN :
+                    if event.key == K_ESCAPE:
                         return False
-                    elif event.type == pygame.MOUSEBUTTONUP:
-                        pos = pygame.mouse.get_pos()
-                        if self.view_game.quit_btn.isClicked(pos):
-                            return False
-                    else:
-                        human.chooseAction()
+
+                    elif event.key == K_1 or event.key == K_KP1 :
```

Comment:
`event.key in [K_1, K_KP1]` -> Simpler and easier to add other conditions.

Reply:
done